AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -414,48 +434,129 @@ public async Task<ActionResult> GetHearingById(Guid hearingId)
+                    BookingsApi.Contract.Requests.Enums.UpdateBookingStatus.Created)
                 {
                     // Set the booking status to failed as the video api failed
                     await UpdateFailedBookingStatus(hearingId);
 
-                    return Ok(new UpdateBookingStatusResponse { Success = false, Message = errorMessage });
+                    return Ok(new UpdateBookingStatusResponse {Success = false, Message = errorMessage});
                 }
+
                 if (ex is BookingsApiException)
                 {
                     var e = ex as BookingsApiException;
-                    if (e.StatusCode == (int)HttpStatusCode.BadRequest) return BadRequest(e.Response);
-                    if (e.StatusCode == (int)HttpStatusCode.NotFound) return NotFound(e.Response);
+                    if (e.StatusCode == (int) HttpStatusCode.BadRequest) return BadRequest(e.Response);
+                    if (e.StatusCode == (int) HttpStatusCode.NotFound) return NotFound(e.Response);
                     return BadRequest(e);
                 }
+
                 return BadRequest(ex.Message);
             }
         }
 
+        /// <summary>
+        ///     Get the conference status.
+        /// </summary>
+        /// <param name="hearingId">The hearing id</param>
+        /// <returns>Success status</returns>
+        [HttpGet("{hearingId}/conference-status")]
+        [SwaggerOperation(OperationId = "GetHearingConferenceStatus")]
+        [ProducesResponseType(typeof(UpdateBookingStatusResponse), (int) HttpStatusCode.OK)]
+        [ProducesResponseType((int) HttpStatusCode.NotFound)]
+        [ProducesResponseType((int) HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> GetHearingConferenceStatus(Guid hearingId)
+        {
+            var errorMessage =
+                $"Failed to get the conference from video api, possibly the conference was not created or the kinly meeting room is null - hearingId: {hearingId}";
+            try
+            {
+                _logger.LogDebug($"Hearing {hearingId} is confirmed. Polling for Conference in VideoApi");
+                var conferenceDetailsResponse =
+                    await _conferenceDetailsService.GetConferenceDetailsByHearingId(hearingId);
+                _logger.LogInformation($"Found conference for hearing {hearingId}");
+                if (conferenceDetailsResponse.HasValidMeetingRoom())
+                {
+                    return Ok(new UpdateBookingStatusResponse
+                    {
+                        Success = true,
+                        TelephoneConferenceId = conferenceDetailsResponse.MeetingRoom.TelephoneConferenceId
+                    });
+                }
+                else
+                {
+                    return Ok(new UpdateBookingStatusResponse {Success = false, Message = errorMessage});
+                }
+            }
+            catch (VideoApiException ex)
```

Comment:
what happens if the hearing doesn't exist? Could we add a test to assert that it returns a 404/not found?